events_protocol/core/logging/__init__.py

```diff
@@ -51,19 +51,27 @@ def __init__(self, klass=None):
     def log(self, level, msg, *args, **kwargs):
         if self.isEnabledFor(level):
             event_context = EventContextHolder.get()
-            _msg = dict(
-                severity=logging.getLevelName(level),
-                logmessage=msg,
+            event_payload = dict(
                 EventID=event_context.id,
                 FlowID=event_context.flow_id,
                 UserId=event_context.user_id,
                 UserType=event_context.user_type,
                 Operation="{}:v{}".format(event_context.event_name, event_context.event_version),
                 logger=self.klass,
                 LoggerName=self.logger.name,
-                logdate=dt.utcnow().isoformat(),
                 ApplicationVersion=self.version,
             )
+            event_payload = dict(filter(lambda item: item[1] is not None, event_payload.items()))
+            event_payload = dict(
+                filter(lambda item: item[1] != "None:vNone", event_payload.items())
+            )
```

Comment:
Essa parte gerou um pouquinho de estranhamento pra mim, pessoal.

Eu entendo que nem todos os atributos do event_payload podem estar preenchidos, mas por que é possível que o `Operation` esteja vazio? Ou seja, que evento pode disparar um log da protocolo de eventos sem ter nome e versão?

Reply:
Não é a chamada do evento em si, mas o registro dele ao criar o App. Nesse momento ele emite uma mensagem onde o evento não é informado.

`{"severity": "INFO", "logmessage": "Registering event handler for get:feature:all V2", "EventID": null, "FlowID": null, "UserId": null, "UserType": null, "Operation": "None:vNone", "logger": "None", "LoggerName": "gb.events_protocol", "logdate": "2022-05-09T19:20:57.343860", "ApplicationVersion": "0.9.2"}`

Ainda não vi outras ocorrências desse formato.